test/controllers/neighborhoods_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class NeighborhoodsControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
```

Comment:
I tend to get rid of any auto-generated files that I don't use. But feel free to leave them in if you think you might fill out the tests later on (although in that case you may want to add `rspec-rails` to your Gemfile so that Rails generates RSpec specs instead of the default Rails tests).

Reply:
Ah!  I didn't even realize they weren't RSpec tests.  I'll remove them.